app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -163,28 +165,58 @@ class SettingsFragment : Fragment(), SettingsView, AppLanguageView {
 
     private fun changeLanguage() {
         context?.let {
+            // Add [String] for locales without specified countries, and
+            // add [Pair] for locales with specified countries.
+            val locales = arrayOf(
```

Comment:
I think `locales` could be a constant...
We can make it a `const` later.

Reply:
Yes. That's a great idea. But how are we going to define a constant of non-primitives? Shall we convert every `Pair` to `String` separated by a delimiter? Like for example, `Pair("pt", "BR")` will be `"pt,BR"` and later split it below in the code? or is there a better way? And if yes, what is the best delimiter to use? A comma? Also shall we define it outside the class at the very top or in a `companion object`? I'm still learning Kotlin and would like to know what the best practices are when we need to create a constant of non-primitives in Kotlin.

Reply:
I have refactored the code and added all the locales in a `string-array`. I feel it's neater that way. If you find the old approach better, just let me know, and I will undo the commit.

Reply:
@HusseinElFeky You can put a constant on the top side of a class (look for some examples of it throughout our code as well as for discussions/recommendations on the web - there are good sources here).

Generally we make use of the strings.xml file(s) to things that will be displayed for the user with optional text styling and formatting - what is not the case.

The comma delimiter is ok.